src/navigation/_CollapsingNavigationMenu.jsx

```diff
@@ -120,7 +120,8 @@ class CollapsingNavigationMenu extends React.Component {
 
   componentDidUpdate() {
     const { isNewSelectedPath } = this.state;
-    if (isNewSelectedPath && this.selectedItem && this.selectedItem.current) {
+    const currentItemPosition = this.selectedItem.current ? this.selectedItem.current.getBoundingClientRect().top : null;
```

Comment:
should we check `this.selectedItem`  before ` this.selectedItem.current`?

Reply:
Added here https://github.com/cerner/terra-dev-site/pull/258/commits/a7ff817b8e4e6b07aff9091744bd1f63af7b23ce

Reply:
@pranav300 This still isn't working 100%. If you keep searching for pages in the search, it often times does not move the nav into view if the tab was previously out of view.

Reply:
Updated it here https://github.com/cerner/terra-dev-site/pull/258/commits/4845880f5089b8bf8e1301dc8164aeb2da29c968.

Now using `navigationMenuHeight` instead of the `window.screen.height`.